### PR TITLE
MDEXP-267: add progress object to jobExecution when uploading an empty file

### PR DESCRIPTION
**Pull request by IlliaDaliek**
###**PURPOSE**
Add progress object to jobExecution when uploading an empty file.
Jira: https://issues.folio.org/browse/MDEXP-267

### CODE REVIEW

**Comment by ViktorSoroka07** [comment]
Please name commits with placing the number of ticket e.g `MDEXP-267: ` in the beginning so they propagate to JIRA.

**Comment by KVupp** [comment]
> Please name commits with placing the number of ticket e.g `MDEXP-267: ` in the beginning so they propagate to JIRA.

@ViktorSoroka07, In backend we squash and merge commits, so we **don't need** to prefix the ticket number of each commit. We just need the PR needs to be prefixed with ticket number, and the final merge commit to contain it.

File: src/test/java/org/folio/rest/impl/DataExportTest.java [hunk 1]
```diff
@@ -86,6 +90,26 @@ public DataExportTest() {
...
+    // when
+    ExportRequest exportRequest = buildExportRequest(uploadedFileDefinition);
+    postRequest(JsonObject.mapFrom(exportRequest), EXPORT_URL);
+    String jobExecutionId = uploadedFileDefinition.getJobExecutionId();
+    // then
+    vertx.setTimer(TIMER_DELAY, handler ->
+      jobExecutionDao.getById(jobExecutionId, tenantId).onSuccess(optionalJobExecution -> {
+        JobExecution jobExecution = optionalJobExecution.get();
+        context.verify(() -> {
+          assertJobExecution(jobExecution, FAIL, EXPORTED_RECORDS_EMPTY);
```
Comment (KVupp) [inline src/test/java/org/folio/rest/impl/DataExportTest.java:107]: Why is it asserted to null value? It must be 0, if you are creating a new Progress Object

**Comment by ViktorSoroka07** [comment]
> @ViktorSoroka07, In backend we squash and merge commits, so we don't need to prefix the ticket number of each commit. We just need the PR needs to be prefixed with ticket number, and the final merge commits to contain it.

At least one commit should be with the ticket number while creating the PR, so there is a link in JIRA to the PR itself. I find it inconvenient when I see when the ticket is in code review status and then heading over the JIRA don't see the link to the PR. In UI we name every commit with the prefix (it does not really hurt) and we squash them too.

File: src/test/java/org/folio/rest/impl/DataExportTest.java (hunk 1 above)
Comment (andrii-novytskyi) [reply src/test/java/org/folio/rest/impl/DataExportTest.java:107]: In my opinion, Progress should be created as a new object, and all fields should be 0

**Comment by andrii-novytskyi** [comment]
@IlliaDaliek Please, upgrade the test coverage at least to 80% is possible

File: src/test/java/org/folio/rest/impl/DataExportTest.java (hunk 1 above)
Comment (IlliaDaliek) [reply src/test/java/org/folio/rest/impl/DataExportTest.java:115]: Constant field value has been changed to 0 value

**Review by illia-borysenko — APPROVED** [review]

File: src/test/java/org/folio/rest/impl/DataExportTest.java (hunk 1 above)
Comment (andrii-novytskyi) [reply src/test/java/org/folio/rest/impl/DataExportTest.java:115]: @IlliaDaliek could you please test if csv file is empty

**Review by andrii-novytskyi — APPROVED** [review]

**Pull request merged**